<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>PSDDialogWidget</class>
  <widget class="QWidget" name="PSDDialogWidget" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>582</width>
        <height>536</height>
      </rect>
    </property>
    <layout class="QGridLayout" >
      <property name="margin" >
        <number>0</number>
      </property>
      <property name="spacing" >
        <number>0</number>
      </property>
      <item row="1" column="0" >
        <widget class="KstFFTOptions" name="_kstFFTOptions" />
      </item>
      <item row="3" column="0" >
        <widget class="CurvePlacementWidget" name="_curvePlacement" />
      </item>
      <item row="2" column="0" >
        <widget class="CurveAppearanceWidget" name="_curveAppearance" />
      </item>
      <item row="0" column="0" >
        <widget class="QGroupBox" name="GroupBox9_3" >
          <property name="sizePolicy" >
            <sizepolicy>
              <hsizetype>5</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="title" >
            <string>Curve Contents</string>
          </property>
          <layout class="QGridLayout" >
            <property name="margin" >
              <number>0</number>
            </property>
            <item row="0" column="0" >
              <widget class="QLabel" name="TextLabel1_6_2_4" >
                <property name="sizePolicy" >
                  <sizepolicy>
                    <hsizetype>0</hsizetype>
                    <vsizetype>0</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="text" >
                  <string>Data vector:</string>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="0" column="1" >
              <widget class="VectorSelector" name="_vector" />
            </item>
          </layout>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <tabstops>
    <tabstop>_vector</tabstop>
    <tabstop>_kstFFTOptions</tabstop>
    <tabstop>_curveAppearance</tabstop>
    <tabstop>_curvePlacement</tabstop>
  </tabstops>
</ui>
